wormhole-connect/src/views/WalletModal.tsx

```diff
@@ -145,6 +145,14 @@ const getWalletOptions = async (
       [Context.SEI]: seiOptions,
       [Context.COSMOS]: wallets.cosmos,
     };
+    // filter allWallets that are not supported by network list config
+    const networkContext = CHAINS_ARR.map((chain) => chain.context);
+    const set = new Set(networkContext);
+    Object.keys(allWallets).forEach((context) => {
+      if (!set.has(context as Context)) {
+        delete allWallets[context as Context];
```

Comment:
Could we avoid any future side effects by deleting and, instead of that, create `allWallets` from the beginning with the set of wallets that we plan to support based on the configs?

Reply:
Ok I update the solution without the "delete" tell me what you think